src/main/resources/swagger/api-docs

```diff
@@ -0,0 +1,29 @@
+{
+  "basePath": "https://rawls-dev.broadinstitute.org",
```

Comment:
Couldn't get relative paths to completely work, but minimized the amount of changes needed when running rawls locally. Just switch basePath to localhost. Is this acceptable?